core/docs/spark-qualification-tool.md

```diff
@@ -503,166 +404,139 @@ println(summaryOutput)
+1. `Estimated GPU Duration`: the predicted runtime of the app if it was run on GPU. The sum of the accelerated operator durations, ML function durations (if applicable), and the durations of anything that could not run on GPU because they are unsupported operators or not SQL/Dataframe.
+2. `Estimated Speed-up`: approximately how much faster the application would be on GPU using the original CPU duration of the app divided by the _Estimated GPU Duration_.
+
+The lower the estimated GPU duration, the higher the `Estimated Speed-up`.
+The processed applications or queries are sorted by the `Estimated Speed-up` and then classified into the following categories:
 
 - `Strongly Recommended`
 - `Recommended`
 - `Not Recommended`
 - `Not Applicable`: indicates that the app has job or stage failures.
 
-As mentioned before, the tool does not guarantee the applications or queries with the highest _recommendation_ will actually be
-accelerated the most. Please refer to [Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md) section.
+As mentioned above, queries and and applications _recommended_ for GPU acceleration are not guaranteed to see acceleration. The estimates are based on the "_SparkPlan_" or "_Executor Nodes_" used in the Spark application and currently supported by the RAPIDS Accelerator for Apache Spark. Not all expressions and datatypes are currently supported, please refer to [Supported Operators](https://github.com/NVIDIA/spark-rapids/blob/main/docs/supported_ops.md) for the latest type and expression support information.
 
-In addition to the _recommendation_, the Qualification tool reports a set of metrics in tasks of SQL Dataframe operations
-within the scope of: "_Entire App_"; "_Stages_"; and "_Execs_". The report is divided into three main levels. The fields
+In addition to the _recommendation_, the Qualification tool reports a set of metrics in the tasks of SQL Dataframe operations
+within the scope of the Application, Stages, and Execs. The report is divided into three main levels. The fields
 of each level are described in details in the following sections: [Detailed App Report](#detailed-app-report),
 [Stages report](#stages-report), and [Execs report](#execs-report). Then we describe the output formats and their file
 locations in [Output Formats](#output-formats) section.
 
 There is an option to print a report at the SQL query level in addition to the application level.
 
-### Detailed App report
+- [Application Report](#application-report)
+- [Stages Report](#stages-report)
+- [Execs Report](#execs-report)
+- [MLFunctions Report](#mlfunctions-report)
+- [MLFunctions Total Duration Report](#mlfunctions-total-duration-report)
+- [HTML Report](#html-report)
+   - [Application Recommendations Summary](#application-recommendations-summary)
+   - [App Details View](#app-details-view)
+   - [Raw Data](#raw-data)
+   - [Per SQL Data](#per-sql-data)
```

Comment:
I don't recommend having TOCs in the middle of the file.
I don't think anyone will remember to update this TOC as the sections change.